### ISSUE TITLE
Sync: it doesnt tells device needs Internet (mobile/desktop)

### ISSUE THREAD

**Issue by DaniellMesquita** [brave/browser-android-tabs#2425]

**Comment by srirambv** [comment]
@DaniellMesquita please add details while logging issues. Just a title doesn't help in most cases. Can't determine if its a bug or feature request.

**Comment by DaniellMesquita** [comment]
Its a feature request, to tell user to connect Internet when it isnt, right when user is trying to sync.

I were 15 minutes trying to sync with a loading message, for then seeing it isnt connected. Would be useful to tell it.

**Comment by srirambv** [comment]
> to tell user to connect Internet when it isnt, right when user is trying to sync.

If there is a network drop while sync is in progress, it will retrigger sync when the device comes online. So this use case would be invalid as there is no manual trigger for sync. 

> I were 15 minutes trying to sync with a loading message, for then seeing it isnt connected.  

Where are you seeing the loading message? Is it on the sync chain creation screen? Can you share screenshot of where you are seeing this message?

**Comment by DaniellMesquita** [comment]
Yes, I refer to sync creation screen. It should tell when user isnt connected.
